project/web/balancer.sls

```diff
@@ -170,6 +170,7 @@ install_letsencrypt:
   git.latest:
     - name: https://github.com/letsencrypt/letsencrypt/
     - target: {{ letsencrypt_dir }}
+    - force_checkout: True
```

Comment:
The problem is that git.latest is doing two things: checkout then pull. The branch is already checked-out on the server, so I think `force_checkout` would only help if we were trying to switch from one branch to another. Since it's failing on the 'pull', I'm hoping that 'force_reset' will actually do `git reset --hard`